stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/TextInputHandler.kt

```diff
@@ -0,0 +1,32 @@
+package io.getstream.chat.android.ui.textinput
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.EditText
+
+public class TextInputHandler(
+    editText: EditText,
+    public var containsTextListener: () -> Unit = {},
+    public var emptyTextListener: () -> Unit = {}
+) {
```

Comment:
This class is not doing much ATM, but I suppose more logic will be added here like "commands" and "mentions", right?

Reply:
That's what I believe will happen, yes. Maybe we want to show a preview of a reaction, like Slack does, or show available commands/users to be mention, like you said.